src/configure

```diff
@@ -70,6 +70,7 @@ Configuration options:
   --static              Build and link against static libraries [default=no]
   --shared              Build and link against shared libraries [default=no]
   --use-cuda            Build with CUDA [default=yes]
+  --without-cudadecoder Build without CUDA decoder [default=no]
```

Comment:
sorry, I don't like double negatives, can we make this --with-cudadecoder=yes by default?

Reply:
No problem, but with-feature-x and without-feature-x are completely common flags in configures...